src/analysis/special_functions/pow.lean

```diff
@@ -437,6 +480,64 @@ lemma continuous_rpow_of_pos (h : ∀a, 0 < g a) (hf : continuous f) (hg : conti
 
 end prove_rpow_is_continuous
 
+section prove_rpow_is_differentiable
+
+lemma has_deriv_at_rpow_of_pos {x : ℝ} (h : 0 < x) (p : ℝ) :
+  has_deriv_at (λ x, x^p) (p * x^(p-1)) x :=
```

Comment:
We will also need the derivative of `uncurry rpow` some day, though it can go to another PR.

Reply:
Yes, although it would be an `fderiv`. We can have a `deriv` version by expressing the derivative of `(f x)^(g x)` in terms of the derivatives of `f x` and `g x`, though. And yes, it will go to another PR (the current one is what I need for the convexity of `x -> x^p`, to define `L^p` distances on finite products).